### PR TITLE
DevWs - Megaman Ingenium ZLL dimmable lights

### PR DESCRIPTION
**Pull request by Duncankn**

### CODE REVIEW

**Comment by shrakuma** [comment]
Hi @Duncankn
Thank you for working on integrating with our platform. We are reviewing the code and we shall keep you posted here and in the other PRs that you have submitted.

**Comment by shrakuma** [comment]
Hi @Duncankn
Thank you for working on integrating with our platform. We are reviewing the code and we shall keep you posted here and in the other PRs that you have submitted.

**Comment by greens** [comment]
@Duncankn can you switch from spaces to tabs on all of these for the sake of consistency?

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 1]
```diff
@@ -48,6 +48,11 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E14 W op/ch 400lm", deviceJoinName: "IKEA TRADFRI LED Bulb"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI transformer 10W", deviceJoinName: "TRÅDFRI Driver for wireless control 10W"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI transformer 30W", deviceJoinName: "TRADFRI Driver for wireless control 30W"
+        //Megaman INGENIUM ZB
+        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF", outClusters: "0019",manufacturer: "Megaman", model: "ZLL-DimmableLight", deviceJoinName: "INGENIUM ZB Dimmable Light"
```
Comment (shrakuma) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: can you switch from spaces to tabs on all of these for the sake of consistency?

Comment (shrakuma) [reply devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: @Duncankn  also for the next set of fingerprints.

**Review by greens — APPROVED** [review]

**Pull request merged**